data/src/main/java/com/example/data/OLD loadData.java

```diff
@@ -0,0 +1,70 @@
+
+package com.example.data;
+import android.util.Log;
+import com.google.gson.Gson;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+/*
+public class oldloadData {
```

Comment:
The file name should not contain the `SPACE`.

The "OLD loadData" is a wrong file name.

If it is unused, you should remove it before commit it.